Library/Formula/apkpatch.rb

```diff
@@ -0,0 +1,23 @@
+class Apkpatch < Formula
+  desc "a tool for build/merge Android Patch file(.apatch)"
```

Comment:
Please remove the indefinite article: `desc "Tool for build/merge Android Patch files (.apatch)"`